config/config.go

```diff
@@ -850,6 +854,10 @@ func (c *ScrapeConfig) Validate(globalConfig GlobalConfig) error {
 		c.MetricNameValidationScheme = globalConfig.MetricNameValidationScheme
 	}
 
+	if c.ConvertClassicHistogramsToNHCB == nil {
+		c.ConvertClassicHistogramsToNHCB = &globalConfig.ConvertClassicHistogramsToNHCB
```

Comment:
I think I would copy the value to the new pointer so we don't get unexpected race issues e.g.

```suggestion
		globalVal := *globalConfig.ConvertClassicHistogramsToNHCB 
		c.ConvertClassicHistogramsToNHCB = globalVal
```

Reply:
Thanks for pointing this out! Updated